doc/nrf/releases_and_maturity/releases/release-notes-2.7.99-cs2.rst

```diff
@@ -0,0 +1,76 @@
+.. _ncs_release_notes_2799_cs2:
+
+|NCS| v2.7.99-cs2 Release Notes
+###############################
+
+.. contents::
+   :local:
+   :depth: 3
+
+|NCS| v2.7.99-cs2 is a **customer sampling** release, tailored exclusively for participants in the nRF54H20 SoC customer sampling program.
+Do not use this release of |NCS| if you are not participating in the program.
+
+The release is not part of the regular release cycle and is specifically designed to support early adopters working with the nRF54H20 SoC.
+It is intended to provide early access to the latest developments for nRF54H20 SoC, enabling participants to experiment with new features and provide feedback.
+
+The scope of this release is intentionally narrow, concentrating solely on the nRF54H20 DK.
+You can use the latest stable release of |NCS| to work with other boards.
+
+All functionality related to nRF54H20 SoC is considered experimental.
+
+Highlights
+**********
+
+Added the following features as experimental:
+
+* :ref:`peripheral_uart` sample:
+
+  * Added a configuration that allows to build the sample application for the nRF54H20 radio core.
```

Comment:
Do we need to have nRF54H20 in all places here as the introduction already states the scope is solely nRF54H20?

Reply:
Could move the nRF54H20 part a couple of lines up so it's "Added the following features for the nRF54H20 as experimental:"

Though I'm not sure if that should be DK or SoC. And if it is not the same for all entries, then it should be kept in all entries and the DK or SoC part added, as appropriate.

Reply:
as Wille suggested, moved nRF54H20 to the introduction phrase and removed from most of the points to avoid repetition